mitodl/open-discussions
# upgrade some JS dependencies

#### Pre-Flight checklist


#### What are the relevant tickets?

none

#### What's this PR do?

Not too much, this is just the result of running `yarn upgrade`. This just upgrades resolved dependencies, so we _should_ not see any breaking changes in terms of APIs and whatnot.

#### How should this be manually tested?

Make sure that the app is booting and running normally. Generally exercise a few features and make sure everything looks good!

Also if the tests and whatnot are all passing that's a good sign :)
